pkg/api/render.go

```diff
@@ -24,25 +24,25 @@ func (hs *HTTPServer) RenderToPng(c *models.ReqContext) {
 
 	width, err := strconv.Atoi(queryReader.Get("width", "800"))
 	if err != nil {
-		c.Handle(400, "Render parameters error", fmt.Errorf("Cannot parse width as int: %s", err))
+		c.Handle(400, "Render parameters error", fmt.Errorf("cannot parse width as int: %s", err))
```

Comment:
I guess the error is not required to be wrapped here because it's an api response?

Reply:
@papagian Yeah, I can't imagine that any wrapped error will be checked by `c.Handle`.